README.md

```diff
@@ -285,6 +288,26 @@ export default BaseAuthorizer.extend({
 });
 ```
 
+For instance if you wanted to use the popular Json Web Tokens mechanism which involves sending an `Authorization: Bearer {token}` header in each request you could implement the authorizer like the following:
+
+```javascript
+import Ember from 'ember';
+import Base from 'ember-simple-auth/authorizers/base';
+
+const { isEmpty } = Ember;
+
+export default Base.extend({
+  authorize(block) {
+    const accessToken = this.get('session.authenticated.token');
+    if (this.get('session.isAuthenticated') && !isEmpty(accessToken)) {
+      block('Authorization', `Bearer ${accessToken}`);
+    }
+  }
+
+});
```

Comment:
Hum, interesting. You're right, I didn't find it while implementing my own solution, so I just rolled my own. Although there's something curious (I will have to test it to make sure it works) because:

I had to use this
```js
const accessToken = this.get('session.authenticated.token');
```
instead of the default on "oauth2-bearer.js
```js
const accessToken = this.get('session.authenticated.access_token');
```
Otherwise the token would never get sent, because `access_token` would always be undefined